src/dao/abi/index.js

```diff
@@ -1,35 +1,28 @@
 /* eslint-disable */
-function optionalRequire (path:string = '') {
-  let result = {}
-  try {
-    result = require('chronobank-smart-contracts/build/contracts/' + path)
-  } catch (e) {
-    console.warn('contract not found:', path)
-  }
-  return result
-}
+import optionalRequireFactory from 'optional-require'
+const optionalRequire = optionalRequireFactory(require)
```

Comment:
I think this is not work for webpack build, and who `optional-require` will resolve files in `chronobank-smart-contracts/build/contracts/` folder